status.go

```diff
@@ -79,6 +79,18 @@ func Model() string {
 	return strings.TrimSpace(model)
 }
 
+// Firmware returns a firmware description for the PC Engines apu2 (and other PCs), e.g. “coreboot v4.11.0.6 (04/26/2020)”.
+func Firmware() string {
+	vendor := mustReadFile0("/sys/class/dmi/id/bios_vendor")
+	version := mustReadFile0("/sys/class/dmi/id/bios_version")
+	date := mustReadFile0("/sys/class/dmi/id/bios_date")
+	if vendor == "" || version == "" || date == "" {
```

Comment:
Instead of not displaying anything when any part is missing, I think it might be nice to display as much as we can. That way, the code works better on platforms we can’t test yet.

Reply:
Fair point, fully agreed. I will change this accordingly

Reply:
See https://github.com/gokrazy/gokrazy/pull/69/files#diff-9ebb930cab0da6fd9029098cf29ed2d6R82-R99